openshift/07-apicast-cloud-hosted-build-config-policy-disabler.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: apicast-cloud-hosted
+  name: apicast-cloud-hosted-policy-disabler
+spec:
+  nodeSelector: null
+  postCommit:
+    args:
+      - '--test'
+      - '--lazy'
+    command:
+      - bin/apicast
+  output:
+    to:
+      kind: ImageStreamTag
+      name: apicast-cloud-hosted-policy-disabler:latest
+  source:
+    type: Dockerfile
+    dockerfile: |
+      FROM apicast-cloud-hosted:latest
+
+      USER root
+
+      RUN rm -f src/apicast/policy/rate_limit/apicast-policy.json \
+       && rm -f src/apicast/policy/token_introspection/apicast-policy.json
+
+      USER 1001
+
+  strategy:
+    type: Docker
+    dockerStrategy:
+      from:
+        kind: ImageStreamTag
+        name: apicast-cloud-hosted:latest
```

Comment:
I think the end result of this build should end up in that stream:tag and the previous build should be in some other tag/stream.

Reply:
yes sure! we don't need two imagestreams

Reply:
done in 75b91358f6c68284ce47f22d144fc86551addb7a